src/Connection/Manager.php

```diff
@@ -321,6 +324,8 @@ private function switchNodeIfNeeded()
             $this->nodeId
         );
 
+        $previous = null;
```

Comment:
Rationale: that's the name of the parameter in `Exception::__construct()`: https://github.com/php/php-src/blob/c6982995504b6e21e8a5ade29cfb16a55196dc43/Zend/zend_exceptions.c#L772